data/Pakistan/Assembly/ep-popolo-v1.0.json

```diff
@@ -2958,10 +2954,10 @@
           "scheme": "wikidata"
         }
       ],
-      "image": "http://www.na.gov.pk/uploads/images/NA-300%20sMayryam.jpg",
+      "image": "http://www.na.gov.pk/uploads/images/final%20s01.JPG",
```

Comment:
Not quite sure what's going on here, but this image doesn't exist.

Reply:
This turned out to be a problem in the scraper which was causing lots of images to break. Fixed in https://github.com/everypolitician-scrapers/pakistan-national-assembly/commit/51abd21b6be62e6ae0e976b20960adbe26c7aff0.